package/Dockerfile

```diff
@@ -40,7 +40,7 @@ ENV KUSTOMIZE_VERSION v3.5.4
 # System charts minimal version
 ENV CATTLE_FLEET_MIN_VERSION=0.3.500
 ENV CATTLE_RANCHER_OPERATOR_MIN_VERSION=0.1.400
-ENV CATTLE_RANCHER_WEBHOOK_MIN_VERSION=0.1.000
+ENV CATTLE_RANCHER_WEBHOOK_MIN_VERSION=0.1.200-rc1
```

Comment:
You will just need to update to 0.1.200. Any version that is below 0.1.200(including rcs) will be installed/upgrade.

Reply:
fixed!